controllers/user.go

```diff
@@ -509,7 +509,19 @@ func (c *ApiController) SetPassword() {
 		return
 	}
 
-	targetUser.Password = newPassword
+	organization, err := object.GetOrganizationByUser(targetUser)
+	if err != nil {
+		c.ResponseError(err.Error())
+		return
+	}
+
+	if organization == nil {
+		c.ResponseError(fmt.Sprintf(c.T("the organization: %s is not found"), userOwner))
+		return
+	}
+	targetUser.UpdateUserPassword(organization)
+
+	//targetUser.Password = newPassword
```

Comment:
have you change the password to the newer one ?

Reply:
Sorry, You are right. 
It should be like this.
```
	targetUser.Password = newPassword
	targetUser.UpdateUserPassword(organization)
```
😂